tests/test_io.py

```diff
@@ -389,10 +389,37 @@ def test_json():
 
 def test_json_series():
     s = DF["a"]
-    with ensure_clean() as path:
-        check(assert_type(s.to_json(path), None), type(None))
-        check(assert_type(read_json(path, typ="series"), Series), Series)
-    check(assert_type(DF.to_json(), str), str)
```

Comment:
Can you leave these tests in there?  They are testing the `path` argument and also when `orient` is not specified.